person.rb

```diff
@@ -1,7 +1,20 @@
 class Person
+  attr_reader :id, :name, :age
+  attr_writer :name, :age
```

Comment:
- [ Optional ] Well done 👍 . Since `name` and `age` are both `attr_reader` and `attr_writer`, ill just recommend using `attr_accessor` i.e

`attr_accessor :name, :age`
`attr_reader :id`